src/main/java/org/broadinstitute/barclay/help/HelpDoclet.java

```diff
@@ -299,9 +340,28 @@ private void emitOutputFromTemplates (
             /* ------------------------------------------------------------------- */
             /* You should do this ONLY ONCE in the whole application life-cycle:   */
             final Configuration cfg = new Configuration(Configuration.VERSION_2_3_23);
-            cfg.setDirectoryForTemplateLoading(settingsDir); // where the template files come from
             cfg.setObjectWrapper(new DefaultObjectWrapper(Configuration.VERSION_2_3_23));
 
+            // We need to set up a scheme to load our settings from wherever they may live.
+            // This means we need to set up a multi-loader including the classpath and any specified options:
+
+            List<TemplateLoader> templateLoaderList = new ArrayList<>();
+
+            // Only add the settings directory if it's valid:
+            if ( settingsDirIsValid ) {
+                templateLoaderList.add( new FileTemplateLoader(new File(settingsDir.getPath())) );
+            }
+            templateLoaderList.add( new ClassTemplateLoader(getClass(), DEFAULT_SETTINGS_CLASSPATH) );
+
+            // Set the order in which to load the templates
+            // Here, loading order is the order in which the loaders are specified in the array.
+            // That is, we load from file first (if it's valid), then fall back to class path
+            MultiTemplateLoader multiTemplateLoader =
```

Comment:
I'm not sure if we can simplify this template loader code if we go with my suggestion about being explicit about using the fallback, but if we can it would be good.